src/commands/profile.ts

```diff
@@ -16,12 +15,12 @@ const __profile: InteractionHandlerPayloads.GuildChatInputCommand = {
 
         // await results and collect
         const submission = await interaction.awaitModalSubmit({
-            time: 240000,
+            time: ModalTime,
         });
 
         // silly man did not do the submit in time like a mongrel
         if (!submission) {
-            console.log(`Submission canceled due to timeout.`);
+            console.log("Submission canceled due to timeout.");
             return;
         }
         submission.deferUpdate();
```

Comment:
`deferUpdate()` must be awaited as `followUp()` in submission will result in a possible "interaction not replied" error.